config/kernelci.conf

```
@@ -25,8 +25,8 @@ lab_config: shell
 [kcidb]
 
 [test_report]
-email_host: your.host
-email_port: your.port
+# smtp_host: localhost
+# smtp_port: 25
```

```suggestion
smtp_host: localhost
smtp_port: 25
```
These lines should be uncommented.

---

No they need to be in comments as sample values, then users have to explicitly un-comment them to use them.  If this is too confusing we can just drop them from the file.

---

I don't think we should drop them. We can provide default values uncommented for which the code works.
Also, why haven't we added email sender and recipients to the configuration file? Why are they static ones?

---

We can't provide them un-commented as that's something for the user to set up. There may not be an SMTP agent on the host, and the user may not (should not) be sending emails by default without knowing.

The email configuration should be in YAML, but that's a follow-up so at the moment it's hard-coded.

---

OK I've dropped them to avoid confusion.

Putting sample config values in comments is very common practice for other projects but it's not something we've been doing for KernelCI config files until now, so I think we should either add a sample config files will everything in comments or nothing at all (opting for the latter here).

---

Makes sense.